assets/templates/login.html

```diff
@@ -81,6 +81,7 @@ <h1>{{.Title}}</h1>
         </div>
       </main>
     </div>
+    <script src="{{asset .Domain "/scripts/polyfills/promise.js"}}"></script>
```

Comment:
Promise and Fetch needs polyfills only for IE? Maybe we can use a conditional comment to load the polyfills only on IE, like this:

```html
<!--[if IE]>
  <script src="{{asset .Domain "/scripts/polyfills/promise.js"}}"></script>
  <script src="{{asset .Domain "/scripts/polyfills/fetch.js"}}"></script>
<![endif]-->
```

Reply:
Forget that, it looks like IE11 doesn't support conditional comments...

Reply:
Also @clochix has reported some user experimenting the same kind of problem on Android.